src/screens/schedules/components/PendingScheduleModalContent/index.tsx

```diff
@@ -56,6 +58,16 @@ const PendingScheduleModalContent = ({
       <Typography variant="body2">{subject}</Typography>
     </DataContainer>
 
+    <DataContainer>
+      <Label>Assunto</Label>
+      <Typography variant="body2">{topic}</Typography>
```

Comment:
é um campo opcional:

```suggestion
      <Typography variant="body2">{topic || '-'}</Typography>
```